config/develop/bridgeserver2.yaml

```diff
@@ -0,0 +1,17 @@
+template_path: templates/bridgeserver2.yaml
+stack_name: bridgeserver2-develop
+parameters:
+  AppHealthcheckUrl: 'HTTP:80/'
+  AwsAutoScalingMaxSize: '1'
+  AwsAutoScalingMinSize: '1'
+  AwsDefaultVpcId: vpc-9c70bbf9
+  AwsEbHealthReportingSystem: enhanced
+  AwsSnsBounceNotificationEndpoint: bridgepf-develop-bounce-notifications@sagebase.org
```

Comment:
No, because BridgeServer2 is intended to fully replace BridgePF, so I don't believe there is need to create a separate email list.

The only reason I can see using a separate email list is if we want to rename our mailing lists from bridgepf* to bridgeserver2*, which I don't think is necessary.